keps/sig-windows/2258-node-service-log-viewer/README.md

```diff
@@ -237,7 +247,7 @@ Options:
       --since='': Return logs after a specific ISO timestamp or relative date. Only applies to node journal or Get-WinEvent logs.
       --tail=0: Return up to this many lines (not more than 100k) from the end of the log. Only applies to node journal or Get-WinEvent logs.
```

Comment:
Default will be most likely `-1` similarly to `kubectl logs`

Reply:
Fixed.